lighthouse-core/report/html/templates.html

```diff
@@ -562,14 +562,15 @@
     .lh-gauge__label {
       font-size: var(--score-title-font-size);
       line-height: var(--score-title-line-height);
-      margin: calc(0.5 * var(--score-title-line-height));
+      margin-bottom: calc(0.5 * var(--score-title-line-height));
       text-align: center;
       color: black;
     }
 
     .lh-category .lh-gauge__label {
```

Comment:
side note: last year, I probably should have done this more bem-y as `.lh-gauge__label--big` instead of relying on descendent selector, etc.

can you add a todo for that?

Reply:
> .lh-gauge__label--big

@Hoten did you note this anywhere? (can add to e.g.  #8185)

Reply:
Sorry, forgot to push. I made a TODO(#8185) comment and will follow up as clean up.